README.md

```diff
@@ -531,6 +531,134 @@ you will update the above playbook as follows:
       OPENWISP_FIRMWARE_UPGRADER_API: true
 ```
 
+Enabling the radius module
+--------------------------
+
+To enable the radius module you need to set `openwisp2_radius` to `true` in
+your `playbook.yml` file. Here's a short summary of how to do this:
+
+**Step 1**: [Install ansible](#install-ansible)
+
+**Step 2**: [Install this role](#install-this-role)
+
+**Step 3**: [Create inventory file](#create-inventory-file)
+
+**Step 4**: Create a playbook file with following contents:
+
+```yaml
+- hosts: openwisp2
+  become: "{{ become | default('yes') }}"
+  roles:
+    - openwisp.openwisp2
+  vars:
+    openwisp2_radius: true
+    openwisp2_freeradius_install: true
+    # set to false when you don't want to register openwisp-radius
+    # API endpoints.
+    openwisp2_radius_urls: true
+```
+
+**Note:** `openwisp2_freeradius_install` option provides a basic configuration of freeradius for openwisp,
+it sets up the [radius user token mechanism](https://openwisp-radius.readthedocs.io/en/latest/user/api.html#radius-user-token-recommended) if you want to use another mechanism or manage your freeradius separately,
+please disable this option by setting it to `false`.
+
+**Step 5**: [Run the playbook](#run-the-playbook)
+
+When the playbook is done running, if you got no errors you can login at:
+
+    https://openwisp2.mydomain.com/admin
+    username: admin
+    password: admin
+
+**Note:** for more information regarding radius configuration options,
+look for the word "radius" in the
+[Role variables](#role-variables) section of this document.
+
+Deploying the upcoming release of OpenWISP
+==========================================
+
+The following steps will help you set up and install the new version of OpenWISP
+which is not released yet, but ships new modules like
+[OpenWISP Monitoring](https://github.com/openwisp/openwisp-monitoring)
+and [OpenWISP RADIUS](https://github.com/openwisp/openwisp-radius),
+which many users need.
+
+Create a directory for organizing your playbook, roles and collections. In this example,
+`openwisp-dev` is used. Create `roles` and `collections` directories in `~/openwisp-dev`.
+
+```
+mkdir -p ~/openwisp-dev/roles
+mkdir -p ~/openwisp-dev/collections
+```
+
+Change directory to `~/openwisp-dev/` in terminal and create configuration
+and requirement files for Ansible.
+
+```
+cd ~/openwisp-dev/
+touch ansible.cfg
+touch requirements.yml
+```
+
+Setup `roles_path` and `collections_paths` variables in `ansible.cfg` as follows:
+
+```
+[defaults]
+roles_path=~/openwisp-dev/roles
+collections_paths=~/openwisp-dev/collections
+```
+
+Ensure your `requirements.yml` contains following content:
+
+```yml
+---
+roles:
+  - src: https://github.com/openwisp/ansible-openwisp2.git
+    version: dev
+    name: openwisp.openwisp2-dev
```

Comment:
Let's update these instructions so they point to master.

Reply:
I woudl keep the name of the role as `openwisp.openwisp2-dev`